### ISSUE TITLE
Operator / CUDA kernel 实现

### ISSUE THREAD

**Issue by whjthu** [InfiniTensor/InfiniTensor#19]
- [x]        ConvTrans // Shared a base class with Conv? @zhengly123  
- [x]        Transpose, @whjthu 
- [x]        Reshape, flatten, Identity, // Similar to Reshape @wendy12022 
- [x]        G2BMM, GBMML // Rename GBMML as GBMM, @Anmuliar  
- [x]        Activation: relu, tanh. Implement an OP for each activation.  @wanghailu0717 
- [x]        Concat,@wendy12022 
- [x]        Add, Sub, Mul, Div, // 四则运算。如何统一实现 @wendy12022 
- [x]        Softmax, @wanghailu0717 
- [x]        Pow, @wendy12022 
- [x]        Pad,
- [x]        Slice,
- [x]        Split,@wendy12022 
- [x]        Extend,
- [x]        Gather,
- [x]        BatchNorm,
// low priority  
- [x]        ReduceMean,
- [x]        Resize,

**Comment by whjthu** [comment]
这个 issue 里列的所有算子均已实现
